content/en/docs/languages/python/benchmarks.md

```diff
@@ -6,6 +6,15 @@ cSpell:ignore: Elems rrggbbaa
 
 <link rel="stylesheet" href="/css/benchmarks.css">
 
+<style>
+   @media (min-width: 100px) {
+     h3{
+         font-size: 0.9rem;
+         font-weight: bold;
+      }
+    }  
```

Comment:
Can you make the css specific to the benchmark pages so it is not affecting any other page? 

If I understand Problem and solution correctly the titles are too long non-breaking words? Is there a way to force breaking them or change the word breaking behavior?

Reply:
@svrnm i Have made the changes requested by you please review .